fitnessllm_dataplatform/stream/strava/services/etl_interface.py

```
@@ -0,0 +1,230 @@
+                destination=f"dev_strava.{stream}",
+                job_config=bigquery.LoadJobConfig(
+                    write_disposition=bigquery.WriteDisposition.WRITE_APPEND
+                ),
+            )
+            self.insert_metrics(
+                metrics_list=metrics,
+                destination="dev_metrics.metrics",
+                timestamp=timestamp,
+                status=Status.SUCCESS,
+            )
+        except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)